tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode.xmi

```diff
@@ -31,7 +31,6 @@
     <deviceProperties name="CspSubarrayNodeFQDN" description="FQDN of the CSP Subarray Node Tango Device Server.">
```

Comment:
Property name should be CspSubarrayFQDN.

Reply:
Change the description accordingly.

Reply:
Done.